apis/projectcontour/v1/httpproxy.go

```diff
@@ -118,6 +118,11 @@ type HeaderMatchCondition struct {
 	// +optional
 	NotContains string `json:"notcontains,omitempty"`
 
+	// IgnoreCase specifies that string matching should be case insensitive.
+	// Note that this has no effect on the Regex parameter.
+	// +optional
+	IgnoreCase bool `json:"ignoreCase,omitempty"`
+
```

Comment:
should also add this note at the top level `HeaderMatchCondition struct`

Reply:
fixed in 33758a5d52a01e106dbef383f0b49f11b235c0b9